inst/rmd/lua/number-sections.lua

```diff
@@ -0,0 +1,53 @@
+--[[
+number-sections - Number sections like the --number-sections option
+
+# MIT License
+
+Copyright (c) 2020 Atsushi Yasumoto
```

Comment:
We need to add a role `cph` to you at https://github.com/rstudio/rmarkdown/blob/482c5b0fa8143ee778ef81b4392cd19528ae925e/DESCRIPTION#L19

Reply:
Thanks for the notice! Done via https://github.com/rstudio/rmarkdown/pull/1879/commits/1919c559b36bbf1d9d8f297a58a6c67e1338fb1b